server/service/src/requisition/common.rs

```diff
@@ -120,6 +121,7 @@ pub fn check_exceeded_max_orders_for_period(
                 .program_id(EqualFilter::equal_to(input.program_id))
                 .order_type(EqualFilter::equal_to(&order_type.name))
                 .period_id(EqualFilter::equal_to(input.period_id))
+                .name_id(EqualFilter::equal_to(input.other_party_id))
```

Comment:
I think more descriptive variable names would help make small fixes like this much more quick.

I would love to rename `name_id` filter to `other_party_name_id`,  and `other_party_id` to `other_party_name_id`

Reply:
Could create a refactor issue 🤷🏻